package.json

```diff
@@ -14,14 +14,11 @@
     "dist-ci": "npm-run-all dist:* dist:*:*",
     "dist:gulp": "gulp",
     "dist:docs": "gulp docs",
-    "dist:prettier": "prettier ./{dist,docs}/*.{scss,css,json,js} --write",
-    "dist:colors:prettier-sketchpalette": "prettier ./dist/*.sketchpalette --write --parser json",
+    "dist:format": "prettier ./{dist,docs}/*.{js,json,css,scss,sketchpalette} --write",
```

Comment:
feels weird to call prettier when it's not listed in the devDependencies. Thoughts?

Reply:
I'm fine with using it as a transitive dependency. It is preferable to specifying prettier in your package.json and having to keep it in lock-step with the version sewing-kit provides to make sure you don't install two differing versions. I think this is a better solution than avoiding using sewing-kit at all

Reply:
Extra thought: How do we guarantee this runs as the final step when running `yarn run dist`? I think previously it was dumb-luck that the` dist:prettier` was last alphabetically. Would changing the name have an effect?